### ISSUE TITLE
Cannot resolve macbook hostname neither discover macbook services from Linux.

### ISSUE THREAD

**Issue by RomuloPBenedetti** [lathiat/avahi#304]
I can resolve my Fedora machine hostname from macbook but cannot resolve my macbook hostname from Fedora. The mac machine is publishing `_ssh._tcp` service but `avahi-browse -ar` cannot resolve neither `avahi-resolve -n my-mac-hostname.local`

**Comment by evverx** [comment]
Looks like incoming mDNS packets were blocked by the firewall on the Fedora machine but it's hard to tell.

If this issue is still relevant could you open a new issue and attach avahi configs, debug logs (they can be turned on by passing the "--debug" option), pcaps showing mDNS packets sent/received by the Fedora machine and firewall rules affecting port 5353.
